### ISSUE TITLE
Retry often fails

### ISSUE THREAD

**Issue by tomwimmenhove** [tomwimmenhove/subarufobrob#1]
Whenever a packet is 'retried', it seems to fail too often. This might be a timing problem (Maybe the phase is different the second transmission?)

**Comment by ghost** [comment]
The harmonics are nuts on rpitx. This works for me wish I had something that i can just inject raw code without creating file to send.

**Comment by yarikmarar** [comment]
is this possible to transmit on hackrfone ?

**Comment by ghost** [comment]
Yes you can transmit with a hackrf one I use Hackrf_ook. You can find it in github.
